### DISCUSSION TITLE
404오류

### DISCUSSION THREAD

**Discussion by syh-RYAN** [morethanmin/morethan-log#88] [Q&A] [answered]
노션 아이디 입력해서 연결은 된거 같은데
노션에서 글 하나 작성해서 올리고 블로그 들어가서 클릭하면 404오류가 뜹니다.. 어떻게 해야할까요?ㅠ

**Comment by morethanmin** [comment, marked as answer]
안녕하세요! 
아래 이미지 노션 페이지 맞으신가요? 우선 다른 페이지와 다르게 페이지 단위로 생성이 안되어있는 것 같아 보이네요..! 제목말고 내용도 입력하고 다시한번 시도해보시겠어요?

<img width="665" alt="image" src="https://user-images.githubusercontent.com/72514247/214747563-e8884eeb-b7a5-4cb6-aae7-4b0c79e1dfc3.png">

**Reply by syh-RYAN** [reply]
진짜 죄송해요..ㅠ
pageid 이런 형식으로 넣는거 맞나요..??ㅠ
pageId: '63a13479a69f45d39591ab7221a7fee4',
죄송하지만 페이지단위형성을 다시 해봐도 안됩니다..ㅠ 안에 내용도 넣어봤구요!
notion : https://www.notion.so/ryansyh/63a13479a69f45d39591ab7221a7fee4?v=a20ac5dd10634031b32056336aac48a3
https://testlog-com.vercel.app/
test해본다고 notion 아이디만 넣어놨거든요ㅠ

**Reply by morethanmin** [reply]
허허 아닙니다 ㅎㅎ 관심가져주시는것만으로도 감사하죠

복제하신 페이지가 `https://www.notion.so/ryansyh/63a13479a69f45d39591ab7221a7fee4?v=a20ac5dd10634031b32056336aac48a3`라면, NOTION_PAGE_ID 는 `63a13479a69f45d39591ab7221a7fee4` 요게 맞습니다!

해당 값만 넣고 바로 배포하면 작성/수정/삭제 잘 될텐데... 아니면 아래 글 읽어보시면서 차근차근 따라해보시는 것도 괜찮을 것 같아요!!

https://jhk0530.medium.com/notion-blog-with-morethanlog-3f4409e6f613

물론 막히시는 게 있거나 궁금하신 부분 있으시면 편하게 여쭤보셔도 괜찮습니다 👍

**Reply by syh-RYAN** [reply]
앗 해결했어요!!
감사합니다 :)
